packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -206,6 +206,7 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       licenseManagement: `${ENTERPRISE_SEARCH_DOCS}license-management.html`,
       machineLearningStart: `${ELASTICSEARCH_DOCS}nlp-example.html`,
       mailService: `${ENTERPRISE_SEARCH_DOCS}mailer-configuration.html`,
+      playground: `${ENTERPRISE_SEARCH_DOCS}playground.html`,
```

Comment:
@joemcelroy I thought we wanted to have the playground docs be part of the Kibana docs? I'll create a stub page so this PR doesn't break the docs build.

Reply:
yes lets put this in kibana docs. Could you do a stub and il update?

Reply:
https://github.com/elastic/kibana/pull/181266 is open, just need to wait for approval